### PR TITLE
Add sidekiq plugin

### PR DESCRIPTION
**Pull request by littlekbt**

### CODE REVIEW

File: mackerel-plugin-sidekiq/lib/sidekiq.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	return m, nil
+}
+
+// Do the plugin
+func Do() {
+	optHost := flag.String("host", "localhost", "Hostname")
+	optPort := flag.String("port", "6379", "Port")
+	optPassword := flag.String("password", "", "Password")
+	optDB := flag.Int("db", 0, "DB")
+	optTempfile := flag.String("tempfile", "/tmp/mackerel-plugin-sidekiq", "Temp file name")
```
Comment (fand) [inline mackerel-plugin-sidekiq/lib/sidekiq.go]: - Default value of Tempfile should not use `/tmp/`.
  - If you need to dynamically set the default value, use helper.SetTempfileByBasename. (ref. #339)
  - If not, you don't have to set the default. go-mackerel-plugin-helper automatically set the default value. (ref. #301)

File: mackerel-plugin-sidekiq/lib/sidekiq.go [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+	"fmt"
+	"strconv"
+
+	r "github.com/go-redis/redis"
+	mp "github.com/mackerelio/go-mackerel-plugin-helper"
+)
+
+// SidekiqPlugin for fetching metrics
+type SidekiqPlugin struct {
+	Tempfile string
```
Comment (fand) [inline mackerel-plugin-sidekiq/lib/sidekiq.go]: `Tempfile` seems to be unused.

**Comment by littlekbt** [comment]
Thanks review.
I fixed program.

**Comment by fand** [comment]
@littlekbt 
LGTM, thanks!!

**Pull request merged**